index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
+    <title>My_Portfolio</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="header-container">
+            <a href="#" class="logo">My Logo</a>
+            <nav class="nav">
+                <i class="fa fa-bars fa-2x"></i>
+            </nav>
+        </div>
+    </header>
+
+    <main class="main">
+        <section class="section-hero">
+            <div class="hero-container">
+                <h1>I’m Eugene Glad to see you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+                    work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                    contact me.</p>
+                <div class="social-links">
+                    <h2>LETS CONNECT</h2>
+                    <ul>
+                        <li><a href="#"><i class="fa fa-facebook fa-2x"></i></a></li>
+                        <li><a href="#"><i class="fa fa-twitter fa-2x"></i></a></li>
+                        <li><a href="#"><i class="fa fa-github fa-2x"></i></a></li>
+                        <li><a href="#"><i class="fa fa-instagram fa-2x"></i></a></li>
+                        <li><a href="#"><i class="fa fa-linkedin fa-2x"></i></a></li>
+                    </ul>
+
```

Comment:
- [Optional] Kinldy make the color and the sizes of this `<h2>` and the social icons match the figma template. You are expected to stick to design as much as possible.
### Your design
![1](https://user-images.githubusercontent.com/94551484/193346809-e519698b-e5f6-4b55-a65f-f623c5b6feee.png)
### Figma template
![f1](https://user-images.githubusercontent.com/94551484/193346813-1d589319-ddcf-465e-b69d-8451cf96a142.png)